pkg/pronestheus_test.go

```diff
@@ -36,6 +36,10 @@ func TestAllMetrics(t *testing.T) {
 	assert.Contains(t, w.Body.String(), `nest_ambient_temperature_celsius{id="enterprises/PROJECT_ID/devices/DEVICE_ID",label="Custom Name",room="Living Room"} 20.23999`)
 	assert.Contains(t, w.Body.String(), `nest_humidity_percent{id="enterprises/PROJECT_ID/devices/DEVICE_ID",label="Custom Name",room="Living Room"} 57`)
 	assert.Contains(t, w.Body.String(), `nest_heating{id="enterprises/PROJECT_ID/devices/DEVICE_ID",label="Custom Name",room="Living Room"} 0`)
+	assert.Contains(t, w.Body.String(), `nest_mode_heat{id="enterprises/PROJECT_ID/devices/DEVICE_ID",label="Custom-Name",room="Living Room"} 1`)
```

Comment:
Please change the `label` to "Custom Name" to unbreak tests.  Labels are now without the hyphen by default -- see https://github.com/klyubin/pronestheus/pull/4. The same comment applies to the next three lines as well.